sqlx-jiff/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+use serde::{Deserialize, Serialize};
+use std::ops::{Deref, DerefMut};
+
+#[cfg(feature = "postgres")]
+mod postgres;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Timestamp(pub jiff::Timestamp);
```

Comment:
I'm not sure if `(pub jiff::Timestamp)` is better or if I should add a `to_jiff` method + a trait with a `to_sqlx` method, as described in https://github.com/BurntSushi/jiff/issues/50#issuecomment-2253457694.

Seems that the wrapper will always wrap exactly `jiff::Timestamp` and if we `pub` it, both construct and retrieve are convenient.

Reply:
Definitely not `pub` it. And even if you did, we'd still want the extension trait and the `to_jiff` method.

I'd encourage you to check out my libraries. I think there are approximately zero instances of a `pub` field like this. There are use cases for them in library crates (`std` has a few of them), but they are pretty rare.

Reply:
OK. Updated.